### ISSUE TITLE
a python implementation of PyALE

### ISSUE THREAD

**Issue by jdkent** [neurostuff/NiMARE#717]
## Summary
this is another project: https://github.com/DanaJomar/PyALE

## Additional details

- diagnostics
  - percentage of contributions
  - FWE corrections
  - see where all the foci are in the image
  - it is difficult to do slicing (difficult to do in PyALE)

## Next steps
- maybe mention in documentation

**Comment by tsalo** [comment]
Are you sure that is the right repo? It looks like that PyALE creates Accumulated Local Effects plots. Maybe I'm just misunderstanding what those are, but there doesn't seem to be a connection to meta-analysis or neuroimaging.

**Comment by jdkent** [comment]
sorry, that was wrong, updated to the correct one

**Comment by tsalo** [comment]
I agree that we should a link to pyALE in our documentation, assuming that it is going to be maintained for a while. Can you go into more detail about the diagnostics you listed? Are these things that would be good to incorporate into NiMARE?

I believe we have tools already for characterizing the percentage of contributions (e.g., [`Jackknife`](https://nimare.readthedocs.io/en/latest/generated/nimare.diagnostics.Jackknife.html#nimare.diagnostics.Jackknife), [`FocusCounter`](https://nimare.readthedocs.io/en/latest/generated/nimare.diagnostics.FocusCounter.html#nimare.diagnostics.FocusCounter)), and for FWE correction. I think a plotting feature for foci in a Dataset would indeed be useful.

By slicing, do you mean splitting up Datasets, or something else?

**Comment by gerionreimann** [comment]
Further aspects and functions of pyALE
1. Quite easy entry
Code is already done in a jupyter notebook, so that only the input (excel files) and the paths need to be created and defined. The parameters' defaults are already set but can easily changed in the notebook.

2. Different analyses: main effect, probabilisti/subsampled/CV’d ALE, (balanced) contrast, ROI and VOI

3. Contains two different corrections for multiple comparison (cFWE and TFCE)
cFWE is suggested to be used by our best practice guidelines for CBMAs (Müller et al. 2018, Tahmasian et al. 2019)

4. Output are contributions, images, and volumes
Contibutions: a file for each ALE analysis with number of experiments, subjects (including average), cluster (coordinate, size) and contributing experiments (A. total MA in cluster, B. average MA per voxel in cluster, C. relative contribution to cluster ale, D. maximum ale contribution in the cluster (single voxel), E. Sample size)
Images:pngs showing the clusters of ALE, corrections, and foci
Volumes: nifit files of ALE, and corrections, as well as for the foci and Z-statistics

**Comment by tsalo** [comment]
@gerionreimann thank you for expanding on pyALE's capabilities. 

>2. Different analyses: main effect, probabilisti/subsampled/CV’d ALE, (balanced) contrast, ROI and VOI

I'm not familiar with probabilistic ALE. Is there a reference of some kind for it?

**Comment by jdkent** [comment]
this is closed by #1009 (pyALE was replaced by JALE)
